projects/pt1/python/torch_mlir_e2e_test/linalg_on_tensors_backends/cpuprotobackend.py

```diff
@@ -34,6 +34,7 @@ def _build_lowering_pipeline(opts: TestOptions):
         "func.func(linalg-fuse-elementwise-ops)",
         "convert-shape-to-std",
         # Bufferize.
+        "one-shot-bufferize",
         "func.func(scf-bufferize)",
         "func.func(tm-tensor-bufferize)",
```

Comment:
Shouldn't one-shot replace all of these?

Reply:
not all of this, it just do some dep analysis on early stage. AFAIU next bufferization passes converts dialect constructions to buffers, so we still need  them